bin/bootstrax

```diff
@@ -569,6 +589,25 @@ def set_run_state(rd, state, return_new_doc=True, **kwargs):
         projection=bootstrax_projection)
 
 
+def set_status_finished(rd):
+    '''Set the status to ready to upload for datamanager and admix'''
+
+    # Only update the status if it does not exist or if it needs to be uploaded
+    ready_to_upload = {'status': 'eb_ready_to_upload'}
+    if rd.get('status') in [None, 'needs_upload']:
+        run_coll.find_one_and_update(
+            {'_id': rd['_id']},
+            {'$set': ready_to_upload})
+    elif rd.get('status') == ready_to_upload.get('status'):
+        # This is strange, boostrax already finished this run before
+        print('WARNING: bootstax has already marked this run as ready for upload. Doing nothing.')
+        pass
+    else:
+        # Do not override this field for runs already uploaded in admix
+        raise ReferenceError(f'Trying to set set the status {rd.get("status")} to '
```

Comment:
The ReferenceError built-in exception apparently means something complicated about weak references; I had never seen it before. Raising a RuntimeError or a ValueError would be a bit more conventional.

Reply:
I changed it in 641f12d